gensim/models/tfidfmodel.py

```diff
@@ -49,8 +67,8 @@ class TfidfModel(interfaces.TransformationABC):
     Model persistency is achieved via its load/save methods.
     """
 
-    def __init__(self, corpus=None, id2word=None, dictionary=None,
-                 wlocal=utils.identity, wglobal=df2idf, normalize=True):
+    def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.identity,
+                 wglobal=df2idf, normalize=True, smartirs=None):
         """
         Compute tf-idf by multiplying a local component (term frequency) with a
```

Comment:
Can you convert all docstrings in this file to numpy-style, according to my previous comment https://github.com/RaRe-Technologies/gensim/pull/1780#discussion_r157712393